packages/portal/src/components/entity/EntityTable.vue

```diff
@@ -120,18 +187,34 @@
         return `/_api/cache/${this.cacheKey}`;
       },
       cacheKey() {
-        // For organisations, get unlocalised labels, for both English and native.
-        return this.type === 'organisations' ?
-          'collections/organisations' :
-          `${this.$i18n.locale}/collections/${this.type}`;
+        return `${this.$i18n.locale}/collections/${this.type}`;
+      }
+    },
+
+    watch: {
+      '$route.query.filter'() {
+        this.filter = this.$route.query.filter;
+        this.updateRouteQuery({ page: 1 });
+      },
+      '$route.query.page'() {
+        this.currentPage = Number(this.$route?.query?.page) || 1;
+      },
+      '$route.query.sort'() {
+        this.sortBy = this.$route.query.sort.split(' ')[0];
```

Comment:
As the logic for these is duplicated here and in data, could we use a computed getter/setter instead?

Reply:
Yes and that's done.